resources/stacks-form/stacks.js

```diff
@@ -0,0 +1,88 @@
+document.querySelector('input[name="orchestrator"]').toggleAttribute('required');
+document.querySelector('input[name="artifact_store"]').toggleAttribute('required');
```

Comment:
There should always be an `orchestrator` and an `artifact_store` input. ZenML starts with one of each named `default` and the `default` stack has both selected. Since you can't delete components that are part of the active stack and both `orchestrator` and `artifact_store` are required and the form inputs are generated from all available components, there will always be at least one input of each.